main.go

```diff
@@ -70,3 +73,13 @@ func configureGC() {
 		debug.SetGCPercent(20)       // activate more aggressive/frequent GC than the default
 	}()
 }
+
+// Ensure we always have more than 1 OS thread running goroutines, since there are issues with having just 1.
+// (E.g. version check doesn't happen at login time, if have only one go proc. Not sure why that happens if have only one
+// proc. Is presumably due to the high CPU usage we see on login if only 1 CPU, even tho can't see any busy-wait in that code)
```

Comment:
strange indeed.. why is it set to 1 btw? because the system has only 1 core?

Reply:
You mean, why would it be set to 1 before we change it?  Yes, because system has only 1 core.  Go runtime tries to match gomaxprocs to the core count, which makes sense given the way the scheduler works in Go.